backend/app/services/project_service.py

```
@@ -226,6 +472,381 @@ async def _prepare_git_checkout(
+    }
+
+
+def _collect_worktree_project_refs(
+    *,
+    db: Session,
+    user_id: int,
+    client_origin: Optional[str],
+) -> list[dict[str, Any]]:
+    query = db.query(Project).filter(
+        Project.user_id == user_id,
+        Project.is_active == True,
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)